docker-compose.override.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+  firefly-app:
```

Comment:
You can add more volumes here to mount the app files from your host inside the container